app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -451,6 +453,14 @@ class WireActivityViewModel @Inject constructor(
 
                     is ObservePersistentWebSocketConnectionStatusUseCase.Result.Success -> {
                         result.persistentWebSocketStatusListFlow.collect { statuses ->
+                            val usersToObserve = statuses
+                                .filter { !it.isPersistentWebSocketEnabled }
+                                .map { it.userId }
+
+                            notificationManager.observeNotificationsAndCallsWhileRunning(
+                                usersToObserve,
+                                this
+                            ) { call -> openIncomingCall(call.conversationId) }
```

Comment:
This for persistent websocket, sould we call observeNotificationsAndCallsPersistently() here ?

Reply:
this observation is for users that disabled PersistentWebSocket, so we want to observe their message only when the app is running.